AndroidTVGateway/agent/app/src/main/java/org/wso2/androidtv/agent/VideoActivity.java

```diff
@@ -41,13 +47,32 @@ protected void onCreate(Bundle savedInstanceState) {
         String url = getIntent().getStringExtra(TVConstants.MESSAGE);
 
         VideoView videoView = (VideoView) findViewById(R.id.videoView);
-        try {
-            videoView.setVideoURI(Uri.parse(URLDecoder.decode(url, "UTF-8")));
-        } catch (UnsupportedEncodingException e) {
-            Log.e("VideoActivity", "Unable to parse url", e);
-            Toast.makeText(getApplicationContext(), "Unable to play video. " + e.getMessage(), Toast.LENGTH_LONG).show();
-            finish();
+        
+        Log.i("VIDEO URL", " " + url);
```

Comment:
Don't use info logs for debug information. Also use simple class name as the TAG of the log. Format the code and remove unnecessary new lines.